Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.12.8-alpine3.21
+
+ENV PYTHOUNNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` should be an absolute path. Changing it to `/app` instead of `app/` will improve clarity and reliability.